### PR TITLE
feat(alphaTex): allow specifying tempo as a float in a string

### PR DESCRIPTION
**Pull request by jonaro00**
### Proposed changes

Stumbled upon a piece that has a BPM of 112.3(!), so I propose this change where the tempo number can also be a float in a string. (using additional dots and number breaks the parser in edge cases)

### Checklist
- [x] I consent that this change becomes part of alphaTab under it's current or any future open source license
- [x] Changes are implemented
- [x] Existing builds tests pass
- [x] New tests were added

## Further details
- [ ] This is a breaking change
- [x] This change will require update of the documentation/website (or leave it as an easter egg :) )

### CODE REVIEW

**Review by Danielku15 — CHANGES_REQUESTED** [review]
I think it would be more natural for writers to write simply decimals `\tempo 144.22` instead of strings.

You can follow the pattern with allow flags to change the parsing logic: https://github.com/CoderLine/alphaTab/blob/859381839185d1ef2edc0b86b1aefe82e332a05c/src/importer/AlphaTexImporter.ts#L1928-L1930

Then in here you could use a new `_allowFloats` to read digits and points and then use `parseFloat`.

https://github.com/CoderLine/alphaTab/blob/859381839185d1ef2edc0b86b1aefe82e332a05c/src/importer/AlphaTexImporter.ts#L673-L680

**Comment by jonaro00** [comment]
Sounds good! I'll get it done.

File: src/importer/AlphaTexImporter.ts [hunk 1]
```diff
@@ -548,7 +549,7 @@ export class AlphaTexImporter extends ScoreImporter {
             } else if (this._ch === 0x2d /* - */) {
                 // negative number
                 // is number?
-                if (this._allowNegatives && this.isDigit(this._ch)) {
+                if (this._allowNegatives) {
```
Comment (jonaro00) [inline src/importer/AlphaTexImporter.ts:552]: (we know that a dash is a digit if negatives are allowed)
